.authors.yml

```
@@ -2431,3 +2431,13 @@
   github: AniketP04
   num_commits: 1
   first_commit: 2023-10-23 07:46:44
+- name: Schuyler Martin
+  email: schuylermartin45@gmail.com
+  num_commits: 1
+  first_commit: 2023-12-05 10:47:28
+  github: schuylermartin45
+- name: samhaese
```

Got name from signed CLA

```suggestion
- name: Sam Haese
  aliases:
  - samhaese
```